WpNginxConfig.conf

```diff
@@ -0,0 +1,44 @@
+server {
+    listen 443 ssl;
+    listen [::]:443 ssl;
+    include snippets/self-signed.conf;
+    include snippets/ssl-params.conf;
+
+
+    server_name 192.168.50.2;
+    root /var/www/wordpress;
+
+    index index.php index.html index.htm;
+
+    location / {
+        #try_files $uri $uri/ =404;
+         try_files $uri $uri/ /index.php$is_args$args;
+    }
+
+    location ~ \.php$ {
+        include snippets/fastcgi-php.conf;
+        fastcgi_pass unix:/var/run/php/php7.4-fpm.sock;
+    }
+
+    location ~ /\.ht {
+        deny all;
+    }
+
+    location = /favicon.ico { log_not_found off; access_log off; }
+    location = /robots.txt { log_not_found off; access_log off; allow all; }
```

Comment:
how do you understand this line?

Reply:
27 = i think this points the location of favicon.ico (the one thats next the url)
28 = same but with robots.txt (robots txt has to do with server overload)